core/src/main/java/org/frankframework/pipes/AbstractPipe.java

```diff
@@ -269,6 +280,20 @@ public Map<String, PipeForward> getForwards() {
 		return forwards;
 	}
 
+	@Nonnull
+	private Set<Forward> getAllowedForwards() {
+		Class<?> clazz = getClass();
+
+		Set<Forward> forwards = new HashSet<>();
+		while (clazz != null) {
+			forwards.addAll(List.of(clazz.getAnnotationsByType(Forward.class)));
+
+			clazz = clazz.getSuperclass();
+		}
```

Comment:
`AnnotationUtils.findAnnotation` doet dit ook, is iets netter.

Reply:
Een class kan `0..n` forwards hebben. Er zijn deprecated methodes die een lijst met annotaties teruggeven, maar die willen we denk ik liever niet gebruiken. De `findAnnotation` method returned maar 1 annotatie waardoor ik er niks aan heb.